src/api-service/__app__/onefuzzlib/proxy.py

```diff
@@ -165,7 +165,10 @@ def stopping(self) -> None:
             self.stopped()
 
     def stopped(self) -> None:
+        if self.state != VmState.stopped:
```

Comment:
Should `set_state()` be responsible for this "set to `<state>` if not `<state>`" logic? Do we ever _not_ want this?

Reply:
This is already done in set_state, so I removed the additional check here.